src/Elasticsearch.Net/Configuration/RequestConfiguration.cs

```diff
@@ -23,6 +23,12 @@ public interface IRequestConfiguration
 		/// </summary>
 		string ContentType { get; set; }
 
+		/// <summary>
+		/// Associate an Id with this user-initiated task, such that it can be located in the cluster task list.
+		/// <pre>https://www.elastic.co/guide/en/elasticsearch/reference/master/tasks.html#_identifying_running_tasks</pre>
+		/// </summary>
+		string OpaqueId { get; set; }

```

Comment:
Add something to indicate that this is valid only for Elasticsearch 6.2.0+

Reply:
++